phoenix-core-client/src/main/java/org/apache/phoenix/query/QueryConstants.java

```diff
@@ -655,11 +655,11 @@ enum JoinType {INNER, LEFT_OUTER}
             SYSTEM_CDC_STREAM_STATUS_TABLE + "\"(\n" +
             // PK columns
             TABLE_NAME + " VARCHAR NOT NULL," +
-            STREAM_STATUS + " VARCHAR NOT NULL," +
+            STREAM_NAME + " VARCHAR NOT NULL," +
             // Non-PK columns
-            STREAM_NAME + " VARCHAR,\n" +
+            STREAM_STATUS + " VARCHAR,\n" +
```

Comment:
@palashc btw the query pattern is going to be:

```
Step 1: Get the active table stream from the metadata.

SELECT STREAM_NAME FROM SYSTEM.CDC_STREAM_STATUS WHERE TABLE_NAME = <table-name> AND STREAM_STATUS = 'ACTIVE'
```

How would keeping STREAM_NAME help here? I missed this earlier.

Reply:
@virajjasani I changed it because the update pattern is 
1. UPSERT (table_name, stream_name, ENABLING)
2. UPSERT (table_name, stream_name, ENABLED)

Reply:
For querying, since there should only be one stream per table, the query should still be efficient?
SELECT STREAM_NAME FROM SYSTEM.CDC_STREAM_STATUS WHERE TABLE_NAME = <table-name> AND STREAM_STATUS = 'ENABLED'

Reply:
Also for the fact that you can can have multiple streams in the same state, see my latest comment on the corresponding [thread](https://docs.google.com/document/d/157KXONp4ZoWmhUBsy-lK3LNAanCsuJ3cb7qYTmozgZo/edit?disco=AAABZrpxMfI) in the design doc.